core/modules/qdisp/QdispPool.h

```diff
@@ -135,23 +135,10 @@ class QdispPool {
 public:
     typedef std::shared_ptr<QdispPool> Ptr;
 
-    QdispPool() {
-        // Numbers are based on 1200 threads in the _pool. Large results
-        // tend to be slow to give up their threads, thus can't be allowed
-        // to eat up the pool. Bandwidth also makes running many of the
-        // slow queries at the same time a burden on the system.
-        // TODO: Set up thread pool size and queues in configuration. DM-10237
-        _prQueue->addPriQueue(0, 1, 90);  // Highest priority - interactive queries
-        _prQueue->addPriQueue(1, 1, 50);  // Outgoing shared scan queries.
-        _prQueue->addPriQueue(2, 6, 850); // FAST queries (Object table)
-        _prQueue->addPriQueue(3, 7, 250); // MEDIUM queries (Source table)
-        _prQueue->addPriQueue(4, 6, 150); // SLOW queries (Object Extra table)
-        _prQueue->addPriQueue(5, 6, 500); // FAST large results
-        _prQueue->addPriQueue(6, 6, 100); // MEDIUM large results
-        _prQueue->addPriQueue(7, 6, 20);  // Everything else (slow things)
-        // default priority is the lowest priority.
-    }
-
+    QdispPool() { _setup(false); }
+    explicit QdispPool(bool unitTest) { _setup(unitTest); }
```

Comment:
Why not just `QdispPool(bool unitTest = false)` instead of defining two constructors, I think default constructor should always behave as `QdispPool(false)`?